dev/bsc/feeds.yaml

```diff
@@ -941,6 +941,26 @@ DEP-USD:
           id: deapcoin
           currency: USD
 
+FUSE-USD:
+  discrepancy: 1.0
+  precision: 6
```

Comment:
Do we need precision 6 for Fuse?
![image](https://user-images.githubusercontent.com/50644753/147956246-541ea6a2-e767-4b93-9a92-8a41bac6ba36.png)

Reply:
It wasn't specified. I think it is fine to have the maximum precision for this, as it is l2d.